src/Common/src/System/Security/Cryptography/AsnReader.cs

```diff
@@ -1089,6 +1090,45 @@ public bool TryGetPrimitiveBitStringValue(out int unusedBitCount, out ReadOnlyMe
             return read;
         }
 
+        public byte[] ReadBitString(out int unusedBitCount)
+        {
+            return ReadBitString(Asn1Tag.PrimitiveBitString, out unusedBitCount);
+        }
+
+        public byte[] ReadBitString(Asn1Tag expectedTag, out int unusedBitCount)
+        {
+            ReadOnlyMemory<byte> memory;
+
+            if (TryGetPrimitiveBitStringValue(expectedTag, out unusedBitCount, out memory))
```

Comment:
`out ReadOnlyMemory<byte> memory`

Reply:
not sure if this will be available in the scope of `if` only though

Reply:
It technically wouldn't limit the scope, but it felt wrong with it declared as an inline out when it was going to be replaced in the else path.